CONTRIBUTING.md

```diff
@@ -8,8 +8,9 @@ agree to abide by the [Contributor Covenant](http://contributor-covenant.org).
 1. [Fork](https://help.github.com/articles/fork-a-repo/) this project
 2. [Clone](https://help.github.com/articles/fork-a-repo/#step-2-create-a-local-clone-of-your-fork) your forked version `git clone git@github.com:<YOUR-USERNAME>/hacktoberfest-practice.git`
 3. Make changes
-4. [Commit](https://help.github.com/articles/adding-a-file-to-a-repository-using-the-command-line/) your changes (write a short descriptive message of what you have done)
-5. [Push](https://help.github.com/articles/pushing-to-a-remote/) your changes to your forked version
-6. Go to original project on GitHub & Create a [Pull Request](https://help.github.com/articles/about-pull-requests/)
+4. Agree to abide by the [Contributor Covenant Code of Conduct](https://www.contributor-covenant.org/).
```

Comment:
This is already mentioned in line 4 😅 not sure if we should add this.

Reply:
Didn't get you...